.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -29,6 +29,7 @@
                 - ⚠️ 8.0: [dotnet-dotnet](https://dev.azure.com/dnceng/internal/_build?definitionId=1219) and the `dotnet/dotnet` commit that represents the release
            - [ ] Navigate to the build link and ensure the `PoisonTests` and `SdkContentTests` are passing. Warnings indicate a baseline diff and should be inspected carefully.
```

Comment:
This step isn't applicable for 8.0 anymore as poison leaks will now cause test failures and the SdkContentTests are in a separate build.
```suggestion
           - [ ] ⚠️ 6.0 / 7.0: Navigate to the build link and ensure the `PoisonTests` and `SdkContentTests` are passing. Warnings indicate a baseline diff and should be inspected carefully.
```